docs/source/notebooks/02_Extract_geographical_watershed_properties.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>Probably good to add a legend with the name of the categories.<em> </em>If you feel fancy, use the same color code for the bars in the first graph as in the second graph.</p><p> </p><pre class="ql-syntax" spellcheck="false">from matplotlib.colors import Normalize
norm = Normalize()
norm.autoscale(unique[:-1])
cm = plt.cm.get_cmap("tab20")
plt.bar(unique[:-1], counts[:-1], colors=cm(norm(unique[:-1])))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>After tinkering with this for a long time, I was able to add your code here but not the legend. Unless I fix the categories, the number of "unique" and "counts" can vary and will not fit with a fixed legend. So instead, I added a cell with an explanation of the categories and I "beautified" the output of land-use types and counts.</p>